docker-compose.yml

```diff
@@ -79,6 +81,41 @@ services:
       # webhook alert target
       - ct-test-srv
 
+  grafana:
+    image: grafana/grafana:latest
+    env_file:
+      - ./test/config/grafana/provisioning/grafana.env
+    ports:
+      # Grafana UI
+      - 3000:3000
+    volumes:
+        - ./test/config/grafana/provisioning/:/etc/grafana/provisioning/
+        - grafana_data:/var/lib/grafana
+    networks:
+      ctnet:
+        ipv4_address: 10.40.50.6
+        aliases:
+          - grafana
+    depends_on:
+      # Grafana pulls metrics from prometheus
+      - prometheus
+
+  grafana-post-config:
+    image: fedora
```

Comment:
Is this only required to make the Grafana dashboard starred to auto-present post-login?

Pulling in a full Fedora image to run some bash isn't super ideal for people with slow internet (:wave:)

Reply:
That's not a problem at all. I can remove it.

Reply:
Thanks!